infra.bs

```diff
@@ -275,6 +275,20 @@ be reported and addressed.
 
 <h2 id=primitive-data-types>Primitive data types</h2>
 
+<h3 id=nulls>Nulls</h3>
+
+<p>The value null is identical to the JavaScript <b>null</b> value. [[!ECMA-262]]
```

Comment:
I'd rather state what null is about. Maybe something like

> The value null is used to indicate "no value"

with an example that uses it more in context, e.g. return X if Y, otherwise return null.

We could state the JS connection as a separate sentence, like we do for Lists, but I think we should define null independently, instead of layering Infra on top of JS like this.